augly/video/augmenters/ffmpeg/trim.py

```diff
@@ -62,15 +60,17 @@ def add_augmenter(
         elif self.end is None:
             self.end = duration
 
-        video = in_stream.video.trim(start=self.start, end=self.end).setpts(
-            "PTS-STARTPTS"
-        )
-
-        if has_audio_stream(kwargs["video_path"]):
-            audio = in_stream.audio.filter_(
-                "atrim", start=self.start, end=self.end
-            ).filter_("asetpts", "PTS-STARTPTS")
-
-            return ffmpeg.concat(video, audio, v=1, a=1), {}
+        command = [
+            "-y",
+            "-i",
+            video_path,
+            "-vf",
+            f"trim={self.start}:{self.end}," + "setpts=PTS-STARTPTS",
+            "-af",
+            f"atrim={self.start}:{self.end}," + "asetpts=PTS-STARTPTS",
```

Comment:
have you ensured that this works for videos with no audio stream?

Reply:
This works https://github.com/facebookresearch/AugLy/pull/163#discussion_r766861727